examples/llava/qwen2_vl_surgery.py

```diff
@@ -139,7 +143,10 @@ def main(args):
             it will be hardcoded in the `clip_image_build_graph` from `clip.cpp`.
     """
 
-    processor: Qwen2VLProcessor = AutoProcessor.from_pretrained(model_name)
+    if local_model:
+        processor: Qwen2VLProcessor = AutoProcessor.from_pretrained(model_path)
+    else: 
```

Comment:
`148: Trailing whitespace` meaning it's a trailing space at line 148

```suggestion
    else:
```

Reply:
In fairness that's what I figured, but somehow in the GitHub text editor there wasn't a whitespace at the end so that's what confused me so